### ISSUE TITLE
ember-source@1.11.0 gem is not actual 1.11.0

### ISSUE THREAD

**Issue by tricknotes** [emberjs/ember.js#10770]
[ember-source@1.11.0](https://rubygems.org/gems/ember-source/versions/1.11.0) has the following diff with [`ember.debug.js`](http://builds.emberjs.com/tags/v1.11.0/ember.debug.js) (that hosted on emberjs.com) and it causes [this issue](https://github.com/emberjs/ember-rails/issues/464).

It seems the ember-source's one was not applied https://github.com/emberjs/ember.js/pull/10751 .
Could someone rebuild and publish ember-source? (it might be 1.11.0.1 or something)

``` diff
--- /Users/tricknotes/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/ember-source-1.11.0/dist/ember.debug.js	2015-03-29 23:43:41.000000000 +0900
+++ ember.debug.js	2015-03-31 00:19:11.000000000 +0900
@@ -15049,7 +15049,7 @@
     })();
 
     if (!canDefinePropertyOnDOM) {
-      exports.defineProperty = defineProperty = function(obj, keyName, desc) {
+      defineProperty = function(obj, keyName, desc) {
         var isNode;
 
         if (typeof Node === "object") {
@@ -15069,7 +15069,7 @@
   }
 
   if (!hasES5CompliantDefineProperty) {
-    exports.defineProperty = defineProperty = function definePropertyPolyfill(obj, keyName, desc) {
+    defineProperty = function definePropertyPolyfill(obj, keyName, desc) {
       if (!desc.get) { obj[keyName] = desc.value; }
     };
   }
@@ -15078,6 +15078,7 @@
   var canDefineNonEnumerableProperties = hasES5CompliantDefineProperty;
 
   exports.hasES5CompliantDefineProperty = hasES5CompliantDefineProperty;
+  exports.defineProperty = defineProperty;
   exports.hasPropertyAccessors = hasPropertyAccessors;
   exports.canDefineNonEnumerableProperties = canDefineNonEnumerableProperties;
 
@@ -17772,7 +17773,7 @@
 
   var tryFinally;
   if (needsFinallyFix) {
-    exports.tryFinally = tryFinally = function(tryable, finalizer, binding) {
+    tryFinally = function(tryable, finalizer, binding) {
       var result, finalResult, finalError;
 
       binding = binding || this;
@@ -17792,7 +17793,7 @@
       return (finalResult === undefined) ? result : finalResult;
     };
   } else {
-    exports.tryFinally = tryFinally = function(tryable, finalizer, binding) {
+    tryFinally = function(tryable, finalizer, binding) {
       var result, finalResult;
 
       binding = binding || this;
@@ -17848,7 +17849,7 @@
   */
   var tryCatchFinally;
   if (needsFinallyFix) {
-    exports.tryCatchFinally = tryCatchFinally = function(tryable, catchable, finalizer, binding) {
+    tryCatchFinally = function(tryable, catchable, finalizer, binding) {
       var result, finalResult, finalError;
 
       binding = binding || this;
@@ -17870,7 +17871,7 @@
       return (finalResult === undefined) ? result : finalResult;
     };
   } else {
-    exports.tryCatchFinally = tryCatchFinally = function(tryable, catchable, finalizer, binding) {
+    tryCatchFinally = function(tryable, catchable, finalizer, binding) {
       var result, finalResult;
 
       binding = binding || this;
@@ -18068,6 +18069,8 @@
   exports.GUID_KEY = GUID_KEY;
   exports.META_DESC = META_DESC;
   exports.EMPTY_META = EMPTY_META;
+  exports.tryCatchFinally = tryCatchFinally;
+  exports.tryFinally = tryFinally;
 
 });
 enifed('ember-metal/watch_key', ['exports', 'ember-metal/core', 'ember-metal/utils', 'ember-metal/platform/define_property', 'ember-metal/properties'], function (exports, Ember, utils, define_property, properties) {
@@ -33122,7 +33125,7 @@
     }
   });
 
-  exports.NativeArray = NativeArray = NativeArray.without.apply(NativeArray, ignore);
+  NativeArray = NativeArray.without.apply(NativeArray, ignore);
 
   /**
     Creates an `Ember.NativeArray` from an Array like object.
@@ -33179,7 +33182,7 @@
   NativeArray.activate = function() {
     NativeArray.apply(Array.prototype);
 
-    exports.A = A = function(arr) { return arr || []; };
+    A = function(arr) { return arr || []; };
   };
 
   if (Ember['default'].EXTEND_PROTOTYPES === true || Ember['default'].EXTEND_PROTOTYPES.Array) {
@@ -33190,6 +33193,7 @@
   exports['default'] = NativeArray;
 
   exports.A = A;
+  exports.NativeArray = NativeArray;
 
 });
 enifed('ember-runtime/system/object', ['exports', 'ember-runtime/system/core_object', 'ember-runtime/mixins/observable'], function (exports, CoreObject, Observable) {
```

**Comment by rwjblue** [comment]
Ugh, sorry about this.  Thank you for tracking it down.

I'll get it re-published as soon as I'm back at my computer....

**Comment by tricknotes** [comment]
current status?

**Comment by jdurand** [comment]
:+1: just slipped and fell flat on my face on this.

**Comment by awj** [comment]
:+1: also waiting on 1.11 due to this.

**Comment by stephencattaneo** [comment]
also waiting on this.

**Comment by rwjblue** [comment]
Just published ember-source-1.11.0.1, apologies for the delay.

**Comment by tricknotes** [comment]
Thanks :green_heart: 
It works for me :-)
